#!/usr/bin/python3
def raise_exception_msg(message=""):
    value = NameError(message)
    raise value
